### DISCUSSION TITLE
مشکل عجیب

### DISCUSSION THREAD

**Discussion by t-51** [Gozargah/Marzban#412] [General]
سلام ببخشید یک سوال من نصب کردم میخواهم وارد پنل بشم صفحه سه بعدی نمی دونم چی باز میشه عکس نمی شه فرستاد وگر نه میفرستادم مشکل چیه ؟

**Comment by AliCHB** [comment]
> سلام ببخشید یک سوال من نصب کردم میخواهم وارد پنل بشم صفحه سه بعدی نمی دونم چی باز میشه عکس نمی شه فرستاد وگر نه میفرستادم مشکل چیه ؟

اسلش داشبورد رو نمیذاری آخرش. فرمت صحیح:
http://YOUR_SERVER_IP:8000/dashboard/
